Dockerfile

```diff
@@ -198,6 +199,9 @@ ARG USER_GID
 RUN usermod --uid $USER_UID www-data && \
     groupmod --gid $USER_GID www-data
 
+# Setup logrotate cron job
+RUN echo "0 0 * * * /usr/sbin/logrotate /etc/logrotate.conf" > /etc/cron.d/logrotate && \
+    chmod 0644 /etc/cron.d/logrotate
```

Comment:
This will not work as there is no cron daemon in docker containers (and that's better)

That's why I was expecting either to use an external cron (on docker staging VM) or to have Apache rotate it's log by itself (this is a capability found in some logging framework, I don't know if it's the case for apache2.

The first option means you might not be able to resolve this issue, but in the -infrastructure repository, and with a manual deployment.